### ISSUE TITLE
publishToMavenLocal always publish to cache repo

### ISSUE THREAD

**Issue by ethauvin** [cbeust/kobalt#291]
Relevant output from [kobalt-maven-local](https://github.com/ethauvin/kobalt-maven-local):
```
Found META-INF/kobalt-plugin.xml in ..\kobaltBuild\libs\kobalt-maven-local-0.4.0-beta.jar
Parsed plugin XML file, found: Maven Local Repository
Found new plug-in, adding it to pluginInfo: com.beust.kobalt.internal.PluginInfo@4a3631f8
Setting Maven Local Repository path to: m2/repository <<< repoPath() in plugin
Found project example in class class Kobalt4526027557208556951Kt
Final list of repos:
  https://jcenter.bintray.com/
  http://repo1.maven.org/maven2/
  file:/C:/Users/erik/.kobalt/localMavenRepo/
  file:/K:/kotlin/kobalt-maven-local/example/m2/repository/ <<< from plugin
  http://repository.jetbrains.com/all/
Applying plug-in "Java"
Applying plug-in "Maven Local Repository"
Applying plug-in "Apt"
Applying plug-in "JvmCompiler"
Applying plug-in "Packaging"
Applying plug-in "Application"
Applying plug-in "Kobalt"
Applying plug-in "Publish"
Applying plug-in "Kotlin"
...
───── example:publishToMavenLocal
Deploying 4 files to local maven C:\Users\erik\.kobalt\localMavenRepo <<< Wrong!
```
I've highlighted/commented on lines with `<<<`

**Comment by cbeust** [comment]
I don't follow: the "list of repos" is the list where the dependencies are downloaded from. These repos have nothing to do with the local Maven cache that `publishToMavenLocal` publishes to.

This directory is either `~/.kobalt/localMavenRepo` by default or whatever is in `settings.xml`.

**Comment by ethauvin** [comment]
Fixed in 0.867.
